lib/bolt_server/transport_app.rb

```diff
@@ -690,11 +690,23 @@ def make_winrm_target(target_hash)
         Bolt::Util.validate_file('inventory file', context[:config].project.inventory_file)
 
         begin
+          # Set the default puppet_library plugin hook if it has not already been
+          # set
+          context[:config].data['plugin-hooks']['puppet_library'] ||= {
```

Comment:
Think this matches https://github.com/puppetlabs/orchestrator/blob/2019.8.x/src/puppetlabs/orchestrator/project_gather_target_facts.clj#L57